UDP/course_materialized_view.md

```diff
@@ -122,3 +98,17 @@ REFRESH MATERIALIZED view v1_student_current_term_course;
 REFRESH MATERIALIZED view v2_student_current_term_course_assignment_avg;
 REFRESH MATERIALIZED view v3_student_current_term_course_activities;
 ```
+
+## Auto-Refreshed Materialized Views
+Details are here: https://resources.unizin.org/display/UDP/UDP+Context+store
+
+## Athletic student google sheet roster
```

Comment:
More info about the additional step how we connect the athletic student roster data.

Reply:
Thanks for adding the details.